README.md

```diff
@@ -34,7 +34,57 @@ To use MAuth authentication you will need:
     if result.status_code == 200:
         print([r['uuid'] for r in result.json()['studies']])
     print(result.text)
-    
+
+Development
+-----------
+We use [travis](https://travisci.com) for automated CI of the code (and status checks are required to pass prior to PR merges being accepted).  
+We use travis to deploy updated versions to PyPI (only from `master`)
+
+For local development (cross version) we use [tox](http://tox.readthedocs.io/en/latest/) with [pyenv](https://github.com/pyenv/pyenv)    
+
+To setup your environment:
+1. Install Python
+1. Install Pyenv
+   ```bash
+   $ brew update
+   $ brew install pyenv
+   ```
+1. Install Pyenv versions for the Tox Suite
+   ```bash
+   $ pyenv install 2.7.13 3.3.6 3.4.6 3.5.3 3.6.1
+   ```
+1. Install Tox 
+   ```bash
+   $ pip install tox tox-pyenv 
+   ```
+1. Setup the local project versions (one for each env in the `envlist`)
+   ```bash
+    $ pyenv local 2.7.13 3.3.6 3.4.6 3.5.3 3.6.1 
+   ```
+1. Make any changes, update the tests and then run tests with `tox`
+   ```bash
```

Comment:
`$ tox`
(seems obvious but makes it clear for the uninitiated - build virtualenvs and run tests for each)

Reply:
Ok, clarified earlier in README